cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -304,6 +305,14 @@ rules:
       - list
       - get
       - watch
+  - apiGroups:
+      - storage.k8s.io
+    resources:
+      - storageclasses
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
ditto

Reply:
The way the module uses that watch now is to keep whatever resource it fetches from k8s up to date. Every time there's an event on a resource it updates that resources corresponding object: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/rook/rook_cluster.py#L165